QUICKSTART.md

```diff
@@ -196,6 +195,87 @@ if __name__ == "__main__":
 ```
 
 
+## Docker build
+
+This section assumes that you have extended the template in the previous
+section to obtain a `main.py` entrypoint for your Docker image that runs your
+custom analytic.
+
+The snippet below defines a `Dockerfile` that installs the Platform SDK and
+its dependencies in a GPU-enabled Docker image that runs the `main.py`
+entrypoint that you provide. It can be easily extended to include any custom
+installation requirements for your analytic.
+
+```
+# A typical base image for GPU deployments. Others are possible
+FROM nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04
+
+#
+# Your custom installation goes here!
+#
+
+# Install `platform-sdk` and its dependencies
+COPY platform-sdk/ /engine/platform-sdk/
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install \
+        sudo \
+        build-essential \
```

Comment:
I'm not sure what build-essential, pkg-config do for us. I don't think we need `curl` either. I'm not sure if we even need `sudo`. I believe `libcputi-dev` and down we definitely use though, correct?

Reply:
Good point. I believe all can be removed.